src/screens/Events/EventsMap.js

```diff
@@ -169,14 +167,17 @@ class EventsMap extends Component {
 }
 
 const mapStateToProps = state => {
-  let storedEvents = state.events.map(event => ({ ...event }));
+  let storedEvents = state.event.events.map(event => ({ ...event }));
```

Comment:
If we need to have the `event` key after `state`, it feels cleaner if we take care of traversing that at the beginning of `mapStateToProps`, because I could see folks messing this up pretty easily. Something like

```js
const { events, errors } = state.event
```